www/templates/hydrantsHelp.html

```diff
@@ -22,7 +22,8 @@ <h4 class="text-stable">Wie können Wasserentnahmestellen editiert werden?</h4>
             Rot eingezeichnete Markierungen sind Wasserentnahmestellen und Feuerwehren, die von OpenFireMap
             zur Verfügung gestellt werden. Weitere Informationen erhalten Sie dazu im
             <a ng-click="openExternalUrl('http://wiki.openstreetmap.org/wiki/DE:OpenFireMap')"
-		class="hyperlink">OpenFireMap Wiki</a>.
+		class="hyperlink">OpenFireMap Wiki</a>. Unter <a ng-click="openExternalUrl('www.osmhydrant.org/')"
```

Comment:
please modify the url to http://www.osmhydrant.org/ the http:// part is important